### DISCUSSION TITLE
ENV variable definition explanations

### DISCUSSION THREAD

**Discussion by LeveL7LLC** [bigcapitalhq/bigcapital#727] [Ideas]
Hey all! Just getting into BigCapital for the first time and I'm discovering there could be some better clarification on some ENV settings.
The first one I'm struggling with is "MAIL_SECURE". What are the accepted values? Yes? SSL? TLS?
It would be very helpful to have maybe an example file with all the possible values.

Thanks!

**Comment by dumbinacan** [comment]
I'm still not sure. Did you find an answer? I can't find it anywhere in their documentation

**Comment by happnatious1** [comment]
Correct me if I'm wrong but the code looks like MAIL_SECURE is hard coded to true in packages/server/src/common/config/mail.ts


`import { registerAs } from '@nestjs/config';

export default registerAs('mail', () => ({
  host: process.env.MAIL_HOST,
  username: process.env.MAIL_USERNAME,
  password: process.env.MAIL_PASSWORD,
  port: parseInt(process.env.MAIL_PORT, 10),
  secure: process.env.MAIL_SECURE === 'true',
  from: {
    name: process.env.MAIL_FROM_NAME,
    address: process.env.MAIL_FROM_ADDRESS,
  },
}));`
